### PR TITLE
WRO-2579: Update moonstone samples repo with new React style APIs - part 2

### PR DESCRIPTION
**Pull request by adrian-cocoara-lgp**
Enact-DCO-1.0-Signed-off-by: Adrian Cocoara adrian.cocoara@lgepartner.com

Migrate Moonstone samples (pattern-locale-switching -> tutorial-kitten-browser) to new ReactAPI

### CODE REVIEW

File: moonstone/pattern-locale-switching/src/index.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import {createRoot} from 'react-dom';
+import {createRoot} from 'react-dom/client';
```
Comment (daniel-stoian-lgp) [inline moonstone/pattern-locale-switching/src/index.js:1]: nice catch :)

File: moonstone/pattern-virtualgridlist-api/src/components/ImageList/ImageList.js [hunk 2]
```diff
@@ -1,38 +1,30 @@
...
+
+	delete rest.dispatch;
+
+	return (
+		<VirtualGridList
+			{...rest}
+			dataSize={imageitems.length}
+			itemRenderer={renderItem}
+			itemSize={{minHeight: ri.scale(270), minWidth: ri.scale(180)}}
+		/>
```
Comment (daniel-stoian-lgp) [inline moonstone/pattern-virtualgridlist-api/src/components/ImageList/ImageList.js:20]: what happened with the "spacing" prop? :D

Comment (adrian-cocoara-lgp) [reply moonstone/pattern-virtualgridlist-api/src/components/ImageList/ImageList.js:20]: I added it back :D

**Review by daniel-stoian-lgp — APPROVED** [review]
LGTM

**Review by stanca-pop-lgp — APPROVED** [review]
All samples work well. Great job!

**Pull request merged**